pyuvdata/tests/test_uvdata.py

```diff
@@ -2471,3 +2471,42 @@ def test_select_with_ant_str():
     uv2 = uv.select(ant_str=ant_str, inplace=inplace)
     nt.assert_count_equal(uv2.get_antpairs(), ant_pairs)
     nt.assert_count_equal(uv2.get_pols(), uv.get_pols())
+
+def test_juldate2ephem():
```

Comment:
I think this function somehow got rescued in a rebase. It should not exist anymore, please take it out along with it's tests.

Reply:
oops you're right